jupyterhub/values.yaml

```
@@ -357,6 +357,7 @@ ingress:
 cull:
   enabled: true
   users: false
+  namedServers: false
```

The script has "remove named servers", while the user facing z2jh configuration is simply to "cull" it. I dislike that it isn't apparent that named servers are stopped and removed, when they could also for example simply be stopped.

I got to run atm, if we can improve this a bit, which isn't great to change later, I'd be happy to merge.

We could for example rename this and potentially also add another one that allows for only stopping of named servers.

---

```suggestion
  removeNamedServers: false
```

👍 to adding removeNamedServers, since just having `namedServers` here looks like named servers weren't getting culled at all prior to this PR. Since remove is an additional action to culling (which is already working on named servers), including the additional verb in the config parameter makes sense to me.

---

Somehow missed this comment for days. I like the new name. The old one was chosen to try and be consistent with the `users` option, so not a lot of thought went into it.